client/client_test.go

```diff
@@ -1098,10 +1098,14 @@ func testFileOpRmWildcard(t *testing.T, sb integration.Sandbox) {
 	require.Equal(t, true, fi.IsDir())
 
 	_, err = os.Stat(filepath.Join(destDir, "foo/target"))
-	require.Equal(t, true, os.IsNotExist(err))
+	if !os.IsNotExist(err) {
+		t.Errorf("expected %s/foo/target to not exist, got error %v", destDir, err)
```

Comment:
This still seems worse than the multi-line version to me. The weird `Equal, true, os.IsNotExist` seems much less obvious than just `!os.IsNotExist(err)`, sticking to standard Go instead of hiding behavior behind a function seems like a worthwhile goal to me, even if the framework technically allows us to add more context than what was done originally.

Reply:
You don't need `Equal` like the example above.

Reply:
Either way. Why is `require.True(foo)` better than `if foo`? This just feels like more stuff and I don't see the benefit it provides. The if/t.Errorf is clear and people have already learned Go, they shouldn't need to learn a test framework too (even if it is rather obvious what it's doing in this case, it's still something I have to stop and think about until I've seen it enough and it won't look the same across projects, whereas Go syntax is obviously universal in Go projects).

Reply:
It's better because it is consistent with the tests in the repo. No other reason.

Reply:
Does that mean you'd accept a PR to remove it from every test in this repo, because I'll do it :)?

I think this PR provides value as is, and I think the consistency with the vast majority of Go projects is more important than consistency within this project in this particular case.

Reply:
There are more complicated cases in this repo where the comparison and the amount of extra code isn't so obvious to defend.  I have no specific love for `testify/require`, I think it is fine and tests read better and provide better information than without it, as go-check was fine (in moby) before it was replaced because someone had a different taste. First project to use `testify/require` was swarmkit, where it went to containerd and here now. I think multiple switches of test frameworks in moby has been a mistake and don't want to repeat it.

Reply:
Fair enough; I don't really feel comfortable using or introducing the use of more test framework stuff, so I'll close this PR for now and debug locally. At some point I'll write something up on why I think *all* test frameworks are a bad idea. Thanks for your consideration!